internal/repos/perforce.go

```diff
@@ -60,7 +62,23 @@ func (s PerforceSource) ListRepos(ctx context.Context, results chan SourceResult
 			return
 		}
 
-		results <- SourceResult{Source: s, Repo: s.makeRepo(depot)}
+		uuu := url.URL{
+			Scheme: "perforce",
+			Host:   s.config.P4Port,
+			Path:   depot,
+			User:   url.UserPassword(s.config.P4User, s.config.P4Passwd),
+		}
+		p4Url, err := vcs.ParseURL(uuu.String())
```

Comment:
How about just `u`? 😀

```suggestion
		u := url.URL{
			Scheme: "perforce",
			Host:   s.config.P4Port,
			Path:   depot,
			User:   url.UserPassword(s.config.P4User, s.config.P4Passwd),
		}
		p4Url, err := vcs.ParseURL(u.String())
```

Reply:
eh, I dunno. "uuu" was easier to see? I just needed a really temporary variable. :-D